### PR TITLE
Hide unused election params

### PR DESCRIPTION
**Pull request by Tyratox**

### CODE REVIEW

File: docs/messageData.md [hunk 1]
```diff
@@ -1287,6 +1287,10 @@ A member of the LAO can cast a vote by publishing an election/cast_vote message
 election’s channel. Each member may cast multiple votes, only the last one will be counted.
 If write-in is allowed for the election then the vote has to have a write-in.
 
+For the generated vote ids, it has to be made sure that the hash is unique and consistent across all subsystems. The hash is computed based on the list of ballot option indices. To disambiguated between `[1, 0]` and `[10]`, the usage of the comma delimiter `,` in between the indices is mandatory and to make the hash unique, the list of indicies should be sorted in ascending order (from small numbers/indicies to bigger ones, from left to right). Last but not least, it must be made sure that this list does not contain any duplicates, i.e. represents a set of indices. (Each ballot option can only be selected once).
```
Comment (nkcr) [inline docs/messageData.md:1290]: Please format the text so it does not exceed 80 chars in width. It improves the readability. (if you're using VScode the [rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) plugin is your best friend).

Comment (Tyratox) [reply docs/messageData.md]: 👍

File: fe1-web/src/features/evoting/screens/CreateElection.tsx [hunk 2]
```diff
@@ -27,6 +26,9 @@ import { Question } from '../objects';
 
 const DEFAULT_ELECTION_DURATION = 3600;
 
+// for now only plurality voting is supported
```
Comment (Xelowak) [inline fe1-web/src/features/evoting/screens/CreateElection.tsx:29]: You should maybe add a date and your GitHub name there to indicate when it was written, like this :
```suggestion
// for now only plurality voting is supported (2022-03-16, Tyratox)
```

File: fe1-web/src/features/evoting/screens/CreateElection.tsx [hunk 3]
```diff
@@ -89,6 +90,7 @@ const CreateElection = ({ route }: any) => {
     questions.map((item) => ({
       ...item,
       id: Hash.fromStringArray(EventTags.QUESTION, electionId.toString(), item.question).toString(),
+      // for now the write_in feature is disabled
```
Comment (Xelowak) [inline fe1-web/src/features/evoting/screens/CreateElection.tsx:93]: Same here.

**Review by Xelowak — APPROVED** [review]
LGTM ! I just think it could be nice to add a date and a GitHub username when using "for now".

**Pull request merged**